app/build.gradle

```diff
@@ -92,8 +92,32 @@ android {
     testOptions {
         unitTests.returnDefaultValues = true
     }
+    // Exclusions added for Paypal SDK
     packagingOptions {
         pickFirst 'kotlin/**'
+        exclude 'AndroidManifest.xml'
```

Comment:
Still not removed

Reply:
Adding this will remove Manifest when anyone tries to decompile the apk and it doesn't affect the functionality

Reply:
But it is unneeded. Why we need to remove the manifest?

Reply:
Okay will remove this 😅